lib/annotations2triannon/open_annotation.rb

```diff
@@ -27,10 +27,7 @@ def initialize(graph=RDF::Graph.new, id=nil)
       else
         @graph = graph
         raise ArgumentError, 'graph must be an open annotation' unless is_annotation?
-        if id.nil?
-          @id = get_id
-        else
-        end
+        id.nil? ? @id = get_id : @id = id
```

Comment:
Yes!  I know that pattern and I was kicking myself about this!  That's what you get from doing some vim line manipulations without starting from scratch.